doc/installation.md

```diff
@@ -243,7 +243,113 @@ In case you do not remember the token you can show it using the `icingacli`:
 ````
 
 Finally visit Icinga Web 2 in your browser to access the setup wizard and complete the installation:
-`/icingaweb2/setup`.
+`/icingaweb2/setup`, or alternatively, see the  [Installation without wizard](#installation-without-wizard) section below.
+
+
+## <a id="installation-without-wizard"></a> Manual installation without wizard
+
+If you are automating the installation of Icinga Web 2, you may want to skip the wizard and do things yourself. These are the steps you'd need to take (assuming you are using MySQL/MariaDB, if you are using PostgreSQL please adapt accordingly. Note you need to have successfully completed the Icinga 2 installation, installed the Icinga Web 2 packages and all the other steps described above first.
+
+  1. Install PHP dependencies: `php5`, `php5-intl`, `php5-imagick` used by Icinga Web 2.
```

Comment:
We should probably add here the `php5-gd` and `php5-mysql` packages too. What do you think?